classes/Ticket.ts

```diff
@@ -41,6 +44,10 @@ export default class {
     ticket.concessionType = ticketAPIData.concessionType;
     if (ticketAPIData.checkedInAt) {
       ticket.checkedIn = true;
+      ticket.checkedInAt = ticketAPIData.checkedInAt;
```

Comment:
Ok please don't hate me but potentially one thing we could do for cherry on top.
We could make the `checkedInAt` property on this class a date time instance:
```suggestion
      ticket.checkedInAt = DateTime.fromISO(ticketAPIData.checkedInAt);
```
This would mean that if we ever need to use the timestamp anywhere, we don't need to convert it into a DateTime object... Also means that in the other file you changed, we can get rid of the function, and just call `ticket.checkedInAt.toHTTP()` in the HTML 😋

Would need to change the type on line 18 to `checkedInAt?: DateTime`.